plugins/polio/js/src/constants/translations/fr.json

```diff
@@ -328,9 +327,7 @@
     "iaso.polio.label.reasonsNoFingerMarked": "Raisons de non-vaccination",
     "iaso.polio.label.receptionPreAlert": "Pré-alerte",
     "iaso.polio.label.receptionVaccineArrivalReport": "Rapport d'arrivée",
-    "iaso.polio.label.recruitmentSurgeUrl": "Url de la Google sheet de recrutement",
```

Comment:
Shouldn't we keep the translations ? As they will still be used in the campaign history for older campaigns

Reply:
@codeKameleon As I removed all field in the DB, these translation won't be needed anymore even for the older campaigns

Reply:
Theses field would still be in the saved json history dump

Reply:
but from the little they have been used we could probably remove them. 

@codeKameleon Maybe it would make send to keep an dict or an array with all the string used in the history page (old and current field) ? So they don't get removed by mistakes